docs/gettingstarted/query_test_report.rst

```diff
@@ -167,6 +167,73 @@ filter test and report all **FAIL** tests.
 
 .. command-output:: buildtest report --filter tags=tutorials --format name,id,state --failure
 
+Find Tests by Start and End Date(``buildtest report --start --end``)
+--------------------------------------------------------
+
+The ``buildtest report --start`` and ``buildtest report --end`` command can be used to retrieve test records based on start and end date.
+
+Let's take a look at these two example, the first test queries report by filtering by tag name ``state`` and ``name``. The second command
```

Comment:
please remove `tag name` from sentence . It should state "test queries report by filtering by state and name" since we are not filtering by tag so that is not correct.